lib/src/list.dart

```diff
@@ -66,64 +73,73 @@ class RealmList<T extends Object> extends collection.ListBase<T> {
     }
   }
 
-  /// Sets the element at the specified index in the list.
   @override
   void operator []=(int index, T value) {
     RealmListInternal.setValue(handle, realm, index, value);
   }
 
-  /// Clears the collection in memory and the references
-  /// to the objects in this collection in Realm.
+  @override
+  void clear() => realmCore.listClear(this);
 
-  /// Removes all elements from this list.
-  ///
-  /// The length of the list becomes zero.
-  /// If the elements are managed [RealmObject]s, they all remain in the Realm.
   @override
-  void clear() {
-    realmCore.listClear(this);
+  bool get isValid => realmCore.listIsValid(this);
+}
+
+class _UnmanagedRealmList<T extends Object> extends collection.ListBase<T> with RealmEntity implements RealmList<T> {
```

Comment:
I remember we decided to remove underscore from _UnmanagedRealmList.

Reply:
Yes.. apparently I only updated the `ManagedRealmList`. 2s.

Reply:
Fixed